src/Identifier/IdentifierCollection.php

```diff
@@ -18,6 +18,10 @@
 use Cake\Core\App;
 use RuntimeException;
 
+/**
+ * Class IdentifierCollection
```

Comment:
Class name should be removed IMO, just noise.

Reply:
Agree. Please change. 
Added it as placeholder for future use because almost all cake classes have short description in first line.